client/src/partials/budget/new/new_budget.js

```diff
@@ -0,0 +1,368 @@
+angular.module('bhima.controllers')
+.controller('budget.new', [
+  '$q',
+  '$scope', 
+  '$window',
+  '$http',
+  '$translate',
+  'validate',
+  'precision',
+  'messenger',
+  'appstate',
+  'SessionService',
+  'connect',
+  'Upload',
+  function($q, $scope, $window, $http, $translate, validate, precision, messenger, appstate, SessionService, connect, Upload) {
```

Comment:
In #719 we decided to use 'controller as' syntax and `controller.$inject` to manage AngularJS dependency injection.  Could you please rewrite this code to reflect that?